pkg/apis/servicecatalog/doc.go

```diff
@@ -19,5 +19,5 @@ limitations under the License.
 
 // Package servicecatalog contains all of the sources needed to make
 // servicebrokers and their related service objects.
-// +groupName=
+// +groupName=servicecatalog.k8s.io
```

Comment:
just curious - does it make sense for groupName to ever be empty?  If not, can we have the generator throw an error?